client/src/game/ui/drawPip.ts

```diff
@@ -3373,6 +3375,47 @@ export default function drawPip(
   }
 
   const hasCustomFill = customFill !== undefined && customFill !== "";
+  if (secondary === true) {
+    ctx.scale(0.8, 0.8);
+
+    ctx.save();
+    ctx.beginPath();
+    ctx.ellipse(0, 0, 80, 80, 0, 0, 2 * Math.PI);
+    ctx.moveTo(90, 0);
+    ctx.ellipse(0, 0, 90, 90, 0, 0, 2 * Math.PI);
+
+    if (hasCustomFill) {
+      ctx.fillStyle = customFill!;
+    } else if (suit.fill === "multi") {
+      const colors = suit.fillColors;
+      const grad = ctx.createLinearGradient(0, -100, 30, 100);
+      for (let i = 0; i < colors.length; ++i) {
+        grad.addColorStop(i / (colors.length - 1), colors[i]);
+      }
+      ctx.fillStyle = grad;
+    } else {
+      ctx.fillStyle = suit.fill;
+    }
+    // Workaround for Konva.Shape's Context not supporting evenodd
+    // This is used in e.g. Empathy view
+    interface Keyable {
```

Comment:
i think this is better written as
```ts
type Keyable = Record<string, any>
```
correct me if i am wrong

Reply:
You're right. I just don't know any typescript